### ISSUE TITLE
Upgrading EKS through cluster edit didn't upgrade clusters worker nodes

### ISSUE THREAD

**Issue by OndrejKu** [rancher/rancher#26998]
**What kind of request is this (question):**
It doesn't seem that upgrading EKS version would upgrade cluster nodes.

**Steps to reproduce (least amount of steps as possible):**
I upgraded rancher version from 2.2.3 -> 2.4.3 and then kubernetes (1.12.7 -> 1.13.12) on cluster (it's on EKS). On both EKS and Rancher cluster view I can see 1.13.12 but worker nodes remains 1.12.7. 
![Screenshot from 2020-05-08 17-09-08](https://user-images.githubusercontent.com/9321195/81445209-d5e12480-914e-11ea-9b52-0dcc38c87db7.png)
![Screenshot from 2020-05-08 17-08-17](https://user-images.githubusercontent.com/9321195/81445213-d7125180-914e-11ea-9bcb-58a5798ccac8.png)

**Result**
I can't say if it's bug or some kind of miss-configuration. Can you advise please?

**Comment by rancher-max** [comment]
Hi @OndrejKu this current behavior is actually by design. We only upgrade the control plan for you, not the nodes. We have plans to switch to using node groups, which will have that ability. 

You'll also notice this is the case if you were to upgrade the cluster directly in AWS. It will have a message saying to upgrade the nodes directly using CloudFormation.

**Comment by OndrejKu** [comment]
Hi @rancher-max thanks for the help. 

Can you point me in documentation how can I upgrade nodes then? And also is it recommended to first upgrade rancher kubernetes to higher version and then upgrade worker nodes or it doesn't matter?

**Comment by rancher-max** [comment]
https://docs.aws.amazon.com/eks/latest/userguide/update-stack.html

That is correct: first upgrade rancher kubernetes to higher version, then upgrade the worker nodes using CloudFormation in AWS. As an example of what the CloudFormation stack will be named, I have an EKS cluster created in Rancher called "maxeks" and the stack for my worker nodes is therefore called: "maxeks-eks-worker-nodes"

**Comment by OndrejKu** [comment]
Hi @rancher-max I followed that tutorial and I can see it's updated but from some unknown reason nodes don't appear in rancher cluster nodes. What am I missing?

**Comment by OndrejKu** [comment]
OK after whole day of researching I was able to solve it. If anyone would need it in the future, here's what I did. 
If you initiate your EKS cluster by Rancher then you cannot follow exactly this tutorial https://docs.aws.amazon.com/eks/latest/userguide/update-stack.html. The important thing is to preserve the template from previous version of CloudFormation. So instead of choosing Replace Current Template you choose Use Current Template in Update CloudFormation menu.
